nbbuild/templates/common.xml

```diff
@@ -427,7 +427,11 @@
         <fileset dir="${cluster}" id="module.executable.files" includes="${nbm.executable.files}"/>
     </target>
 
-    <target name="nbm" depends="init,netbeans,-nbm-prompt-for-storepass,-init-executables" description="Build NBM archive.">
+    <target name="-init-extra.nbm.files" unless="extra.nbm.files.provided"> <!-- fallback -->
+        <zipfileset dir="${cluster}" id="extra.nbm.files" includes="[NOTHING]"/>
```

Comment:
I don't understand the "[NOTHING]" here - what is the intention?

Reply:
The goal here is to ensure there is a zipfileset "extra.nbm.files" defined, but is empty. There is a similar thing nearby (-init-executables), so I simply copied the includes here. common.xml is used by platform applications as well, so we should not be enforcing core NB constraints here, so I tried to make the changes compatibly.

Reply:
Thanks - that way it makes sense.